src/test/regress/multi_schedule

```diff
@@ -215,7 +215,9 @@ test: multi_transaction_recovery
 # multi_router_planner creates hash partitioned tables.
 # ---------
 test: multi_copy fast_path_router_modify pg_dump
-test: multi_router_planner multi_router_planner_fast_path null_parameters
+test: multi_router_planner
+test: null_parameters
+test: multi_router_planner_fast_path
```

Comment:
Needs a comment to prevent us from parallelizing it in the future: 
```
# This test has prepared statements which sometimes get invalidated by concurrent tests,
# changing the debug output.
```

Reply:
It seems that we re-define the same function `author_articles_max_id` on `multi_router_planner_fast_path.sql` and `multi_mx_router_planner.sql`. That looks the reason for the invalidation.

Should we maybe rename the function in on file not to lose the concurrency in the tests? @SaitTalhaNisanci please  first try if that is the  actual reason.

check-multi is getting  longer and longer, I'd prefer not losing paralleism

Reply:
@onderkalaci we had the same problem with null_parameters.sql test with null_select_on_json_param function. However as far as I can see it is defined only in that file, so I would guess that it may not be the reason. 

this test doesn't fail that often so I don't have an easy way of seeing if the issue will be resolved after trying your suggestion.  (Other than running this on circle ci 10-20 times)

I think for check-multi test we can think of some other solutions, maybe we can split it into 2 schedules? So that it will be faster, because in general as check-multi takes the longest, they are running alone for a long time.

Reply:
> this test doesn't fail that often so I don't have an easy way of seeing if the issue will be resolved after trying your suggestion. (Other than running this on circle ci 10-20 times)

Can't you simply locally try? Does a function re-create invalidates the prepared statements on another session?

Anyway, I don't have strong objections merging like this, I just try to keep this in mind that  test times are getting bigger everyday. But, I  think it'd  be nice to learn the behavior anyway.

Reply:
I can try that locally but I cannot see locally if it will fix the flakiness of this test. I will try to learn the behavior locally.

Reply:
@onderkalaci  It seems that recreating the function in another session doesn't invalidate the cache. 

I have used two sessions, when I recreate the same function from the same session, it invalidates the cache. When I recreate the same function from the other session, it doesn't change anything.  Is this what you wanted me to try?

Reply:
Yes, thanks for testing it